libs/sdk/code-examples-sdk/project.json

```diff
@@ -0,0 +1,69 @@
+{
+  "name": "code-examples-sdk",
```

Comment:
Without it we'd conflict with `code-examples` the SPA. This was my compromise, it could've been a prefix, or it could use `-e2e` but it doesn't work exactly how other `-e2e` projects work.